rdt-open-reader/src/main/java/io/ona/rdt/util/Utils.java

```diff
@@ -99,8 +102,9 @@ public static int convertDpToPixels(Context context, float dp) {
         return (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, context.getResources().getDisplayMetrics());
     }
 
-    public static void updateLocale(android.content.Context context) {
-        Locale locale = new Locale(BuildConfig.LOCALE);
+    public static void updateLocale(android.content.Context context, AllSharedPreferences allSharedPreferences) {
+
+        Locale locale = new Locale(allSharedPreferences.fetchLanguagePreference());
```

Comment:
https://github.com/onaio/rdt-standard/pull/457/files#r478863684. No need to pass the `AllSharedPreferences`.

Reply:
We may also need to have a default fallback Locale that can be defined in the gradle file.

Reply:
done